lib/stdlib/test/qlc_SUITE.erl

```diff
@@ -7257,15 +7257,18 @@ manpage(Config) when is_list(Config) ->
     [2,3,4] = qlc:eval(QH),
 
     %% ets(3)
-    MS = ets:fun2ms(fun({X,Y}) when (X > 1) or (X < 5) -> {Y} end),
```

Comment:
It was both: The old condition seemed odd, but it also broke the test, since the test is asserting the handle is the same via `ets:fun2ms` and `qlc`, but this isn't true if `ets:fun2ms` applies some optimisations (which it did in the original case)